### PR TITLE
Chore: Fix typos

### PR DESCRIPTION
**Pull request by FlorianCassayre**
Corrected a spelling mistake across four files: replaced `recieved` by `received`.

image

### CODE REVIEW

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json [hunk 1]
```diff
@@ -1979,7 +1979,7 @@
               "subscription": ""
             }
           ],
-          "title": "Max Available Bytes Recieved and Trend  Line",
+          "title": "Max Available Bytes Received and Trend  Line",
```
Comment (fridgepoet) [inline public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json:1982]: Thanks! I wonder if that extra space between `Trend` and `Line` was intentional..
```suggestion
          "title": "Max Available Bytes Received and Trend Line",
```

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsWorkspace.json [hunk 2]
```diff
@@ -1970,7 +1970,7 @@
               "subscription": ""
             }
           ],
-          "title": "Max Available Bytes Recieved and Trend  Line",
+          "title": "Max Available Bytes Received and Trend  Line",
```
Comment (fridgepoet) [inline public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsWorkspace.json:1973]:
```suggestion
          "title": "Max Available Bytes Received and Trend Line",
```

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json (hunk 1 above)
Comment (FlorianCassayre) [reply public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json:1982]: Thanks.
I wondered as well; probably not. I'll fix it.

**Review by fridgepoet — APPROVED** [review]

**Review by yaelleC — APPROVED** [review]
Thanks for that @FlorianCassayre 👍 
LGTM

Note: It's possible that some external plugins e2e tests will fail after this if they are asserting that the error message that appears. I ran Grafana's e2e tests locally, they are still passing, and a quick search on GitHub did not show anything.
Just making a note of it in case we come across failing tests soon 😉

**Pull request merged**